db/migrate/20230110174055_create_activity_insight_oa_files.rb

```diff
@@ -0,0 +1,16 @@
+class CreateActivityInsightOaFiles < ActiveRecord::Migration[6.1]
+  def change
+    create_table :activity_insight_oa_files do |t|
+      t.string :location
+      t.string :checksum
+      t.bigint :publication_id, null: false
+
+      t.timestamps
+    end
+
+    add_foreign_key :activity_insight_oa_files, 
+                    :publications, 
+                    name: :activity_insight_oa_files_publication_id_fk,
+                    on_delete: :cascade
+  end
+end
```

Comment:
@ajkiessl I think we should add an index on the foreign key column (`publication_id`) too.